nasa_wildfires/cli.py

```diff
@@ -16,32 +16,38 @@ def cmd():
 @cmd.command(help="Hotspots detected by the MODIS satellite in a recent 24-hour period")
 @click.option('-r', '--region', default="Global", type=click.Choice(
     options.REGION_DICT.keys(), case_sensitive=False), help="Hotspot region")
+@click.option('-t', '--time-range', default="24h", type=click.Choice(
+    options.TIME_RANGE, case_sensitive=False), help="Time range")
 @click.option('--indent', default=0, help='Indentation of output')
 @click.option('--sort-keys/--no-sort-keys', default=True, help="Sort the properties keys")
-def modis(region, indent, sort_keys):
-    data = get_modis(region)
+def modis(region, time_range, indent, sort_keys):
+    data = get_modis(region, time_range)
```

Comment:
If we call it `time_range` here, we should call it `time_range` in the other file, not `time`. Just for consistency.